app/models/current.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+module Current
```

Comment:
This is not a model, nor solely related to models, so does not belong in /app/models.

Perhaps create a new directory /app/utilities and put this in there?